### DISCUSSION TITLE
why ver 1.0.0 can’t support multi level in sidebar

### DISCUSSION THREAD

**Discussion by yizhi996** [vuejs/vitepress#737] [Q&A] [answered]
The 0.x version `sidebar` supported multi level `children`, but 1.0.0 `SidebarItem` without `items` attribute.
@kiaking

**Comment by acbetter** [comment]
same question here, hope it will be supported
```js
function sidebarConfig() {
  return [
    {
      text: 'Config',
      collapsible: true,
      collapsed: true,
      items: [
        { text: 'App Configs', link: '/config/app-configs' },
        { text: 'Theme Configs', link: '/config/theme-configs' },
        { text: 'Frontmatter Configs', link: '/config/frontmatter-configs' },
        {
          text: 'Introduction', collapsible: true,
          collapsed: true,
          items: [
            { text: 'Introduction', link: '/config/introduction' },
            { text: 'App Configs', link: '/config/app-configs' },
            { text: 'Theme Configs', link: '/config/theme-configs' },
            { text: 'Frontmatter Configs', link: '/config/frontmatter-configs' }
          ]
        },
      ]
    }
  ]
}
```

**Comment by kiaking** [comment, marked as answer]
We're planning to support this 👍

**Reply by yizhi996** [reply]
Thanks!

**Reply by erondpowell** [reply]
Awesome news! Is there an estimated timeline on support for nested/collapsible menus?

**Reply by kiaking** [reply]
No I don't have one. But sometime not that far I guess 👀

**Reply by cy920820** [reply]
Thank you. Which version is the plan in

**Reply by Zhengqbbb** [reply]
Our team has needed the features TAT 😭
<img width="270" alt="image" src="https://user-images.githubusercontent.com/40693636/196143639-7b56fadf-3670-4ab1-bd3e-4c14d22a4ecb.png">

**Reply by liuliangsir** [reply]
@Zhengqbbb I find that vitepress@1.0.0-alpha.28 has already supported that feature.

**Reply by Zhengqbbb** [reply]
> @Zhengqbbb I find that vitepress@1.0.0-alpha.28 has already supported that feature.

Not yet. the PR is processing
Ref:
- https://github.com/vuejs/vitepress/pull/1361
- https://github.com/vuejs/vitepress/issues/1360

**Reply by liuliangsir** [reply]
Sorry, I made a mistake, which was that I assumed you needed a support for multi level in sidebar.

**Comment by erondpowell** [comment]
<img width="246" alt="Screenshot 2022-11-21 at 2 01 43 PM" src="https://user-images.githubusercontent.com/62306630/203147822-57bb8cc5-796d-4dce-b758-02c59ba4d893.png">

@liuliangsir Yes defintiely, our project is currently using nested sidebars.

**Reply by goozoon** [reply]
`collapsible: true,` work only for toplevel, not the submenus, right?

**Reply by erondpowell** [reply]
To my understanding, it does not. Though I'm not a vitepress maintainer, and not 100% sure if it does currently or is planned.
